controllers/users.js

```diff
@@ -388,6 +388,7 @@ const getSelfDetails = async (req, res) => {
 
 /**
  * Update the user
+ * @deprecated [SCHEDULED] This Controller will be removed in the near Future, New controller TBD.
```

Comment:
- will you be adding the alternative to this? if not can you please mention the user id of the person you know?

Reply:
no I'll not be adding the alternatives, [vikas](https://github.com/vikasosmium) is working on deprecating /users/self routes

Reply:
@shubhdevelop please add a todo comment with vikas github id stating he would be adding this alternative

Reply:
Added

Reply:
@yesyash Added, todo comments can you please review & approve of the changes

Reply:
I think mentioning the issue ticket here is sufficient.

Reply:
I had added, as asked by yash